src/main/java/Util.java

```diff
@@ -1,6 +1,26 @@
 public class Util {
-    public static void main(String[] arg){
-
+    public static int[] removeElement(int[] array, int t){
+        int Arrlen = 0;
```

Comment:
```suggestion
        int arrLen = 0;
```
nit: camel case should be used